src/components/common/banner/Banner.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from "@material-ui/core";
+import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+
+import classes from "./banner.module.css";
+import Search from "../search/Search";
+import { Dispatch } from "react";
+import ISearchResults from "../search/types/interfaces";
+
+type BannerProps = {
+  setIsSearchOptionSelected: Dispatch<React.SetStateAction<boolean>>;
+  setChosenResult: Dispatch<React.SetStateAction<ISearchResults>>;
+};
+
+const Banner = ({
```

Comment:
What we usually do here is const `Banner: React.FC<BannerProps> = ({...}) =>... `
Please change